src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -254,7 +254,9 @@ class FormPage extends React.Component {
     const showNavLinks =
       environment.isLocalhost() && route.formConfig?.dev?.showNavLinks;
     const hideNavButtons =
-      !environment.isProduction() && route.formConfig?.formOptions?.noBottomNav;
+      !environment.isProduction() &&
+      (route.formConfig?.formOptions?.noBottomNav ||
+        route.pageConfig?.hideNavButtons);
```

Comment:
nice addition, seems reasonable. can you please update `form-system/src/js/types.js` `FormConfigPage` to include `hideNavButtons`?

Reply:
Also note that this will also make it so the action for the logged in user "Save and edit later" type of action, and "Your progress has been saved" type of message won't show up. (that goes in `contentBeforeButtons` and `contentAfterButtons`)